mod-orders/schemas/patch_pieces_request.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Patch pieces request",
+  "type": "object",
+  "properties": {
+    "pieces": {
+      "description": "An array of piece record IDs",
+      "type": "array",
+      "items": {
+        "description": "UUID of an piece record",
+        "$ref": "../../common/schemas/uuid.json"
+      },
+      "uniqueItems": true
+    },
+    "receivingStatus": {
```

Comment:
thinking do we need specify some particular field name here, or use additionalProperies: true like this: https://github.com/folio-org/acq-models/pull/444/files#diff-173686222ddddba2f4eb4fcc727e2cd8015f53861a90f5005de6912182a99f92R38 
In this case our endpoint can be used for patch update of other fields as well